src/controllers/collaboratorController.js

```diff
@@ -0,0 +1,94 @@
+const collaboratorQueries = require("../db/queries.collaborators.js");
+const Authorizer = require("../policies/application");
+const User = require("../db/models").User;
+const Collaborator = require("../db/models").Collaborator;
+const wikiQueries = require("../db/queries.wikis");
+const userQueries = require("../db/queries.users");
+
+module.exports = {
+    addCollaborator(req, res, next){
+            User.findAll({where: {id: req.body.userId}}).then((users) => {
+                users.forEach((user) => {
+                    const authorized = new Authorizer(user)._isPremium();
+
+                    if(authorized) {
```

Comment:
here if you have a list of three users in order [authorized, not authorized, authorized], you're letting the first one in but not the third one. It should either add all the authorized users or none.